console/src/i18n/locales.ts

```diff
@@ -206,6 +222,22 @@ const runtime = {
         en: 'View',
         zh: '查看',
     },
+    'runtime.selector.lastest': {
+        en: 'Recently used',
+        zh: '最近使用',
+    },
+    'runtime.selector.current': {
+        en: 'This project‘s runtimes',
```

Comment:
Change the `‘` https://www.compart.com/en/unicode/U+2018 to the `'` https://www.compart.com/en/unicode/U+0027?

Reply:
done